### ISSUE TITLE
How to create the dataset

### ISSUE THREAD

**Issue by pankaj2701** [jtkim-kaist/VAD#11]
I am wondering how to create the training dataset. I have understood the format but don't know how to create one. Do we have to manually annotate the training dataset. Manually annotation would be difficult, is there any utility which can create an approximate training data which can then refined manually.

**Comment by jtkim-kaist** [comment]
Our provided recorded dataset was manually annotated by two professional engineers. 

If you want to construct the training set, do the followings:

1. run the VAD to the clean speech to get the label.

2. Add the noise to that clean speech using FanT tool or voice box(Matlab implemented)

3. Then you can get the labels with noisy speech

If you don't have clean speech manual annotation cannot be avoidable.
